dapp/src/utils/acreApi.ts

```diff
@@ -82,11 +82,20 @@ const claimPoints = async (address: string) => {
   }
 }
 
+const verifyAccessCode = async (encodedCode: string) => {
+  const response = await axios.post<{ isValid: boolean }>("access/verify", {
+    encodedCode,
+  })
+
+  return response.data
+}
```

Comment:
We could return the boolean result directly instead of returning the data object.

```suggestion
const verifyAccessCode = async (encodedCode: string): Promise<boolean> => {
  const response = await axios.post<{ isValid: boolean }>("access/verify", {
    encodedCode,
  })

  return response.data.isValid
}
```

Reply:
https://github.com/thesis/acre/pull/767/commits/72e5bda6e3375e4cd62b2d0aa44c670de6a049ec